trailblazer/store/filters/analyses_filters.py

```diff
@@ -11,21 +13,45 @@ def filter_analyses_by_id(analyses: Query, analysis_id: int, **kwargs) -> Query:
     return analyses.filter(Analysis.id == analysis_id)
 
 
+def filter_analyses_by_case(analyses: Query, case: str, **kwargs) -> Query:
+    """Filter analyses by case."""
+    return analyses.filter(Analysis.family == case)
```

Comment:
```suggestion
    """Filter analyses by case internal id."""
    return analyses.filter(Analysis.family == case_internal_id)
```

Reply:
Using case_name as there is no internal case id in Traiblazer so it would not make sense and couple to cg too tigthly